src/bulk_sampling.jl

```diff
@@ -1,5 +1,7 @@
 
-using AliasTables
+using BitIntegers, AliasTables
+
+BitIntegers.@define_integers 2232
```

Comment:
Would 2240 be faster because it is a multiple of word size?

Reply:
good idea